openmp/libomptarget/test/unified_shared_memory/close_enter_exit.c

```diff
@@ -5,6 +5,8 @@
 
 // Fails on amdgpu with error: GPU Memory Error
```

Comment:
Fails on [openmp-offload-cuda-runtime](https://lab.llvm.org/staging/#/builders/100) with: "PluginInterface" error: Failure to synchronize stream (nil): Error in cuStreamSynchronize: an illegal memory access was encountered